test/extended/router/http2.go

```diff
@@ -89,7 +90,7 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 	})
 
 	g.Describe("The HAProxy router", func() {
-		g.It("should pass the http2 tests [apigroup:image.openshift.io][apigroup:template.openshift.io]", func() {
+		g.It("should pass the http2 tests [apigroup:image.openshift.io]", func() {
```

Comment:
The fixture in `router-http2-routes.yaml` is still used at line 386 as of this commit, so I think it's premature to remove "[apigroup:template.openshift.io]" here.  It isn't too important if a subsequent commit removes the use of the fixture though.